cluster-up/cluster/kind-k8s-sriov-1.17.0/README.md

```diff
@@ -41,4 +42,32 @@ export KIND_VERSION="0.9.0"
 export KUBECTL_PATH="/usr/bin/kubectl"
 ```
 This allows users to test or use custom images / different kind versions before making them official.
-See https://github.com/kubernetes-sigs/kind/releases for details about node images according to the kind version.
\ No newline at end of file
+See https://github.com/kubernetes-sigs/kind/releases for details about node images according to the kind version.
+
+## Running multi sriov clusters locally
+Kubevirtci sriov provider supports running two clusters side by side with few known limitations.
+
+General considerations:
+
+- A sriov PF must be available for each cluster.
+In order to achieve that, there are two options:
+1. Assign just one PF for each cluster by using `export PF_COUNT_PER_NODE=1` (this is the default value).
+2. Optional method: `PF_BLACKLIST` the non used PFs, in order to prevent them from being allocated to the current cluster.
```

Comment:
What do you think about changing  "`PF_BLACKLIST` the non..." to  `export PF_BLACKLIST=<PF name>` the non..." , like you wrote one line above? 
It will be a bit clearer I think.